modules/nf-core/cobrameta/tests/main.nf.test

```diff
@@ -0,0 +1,123 @@
+nextflow_process {
+
+    name "Test Process COBRAMETA"
+    script "../main.nf"
+    process "COBRAMETA"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "cobrameta"
+    tag "gunzip"
+
+    setup {
+        run("GUNZIP") {
+            script "../../gunzip/main.nf"
+            process {
+                """
+                input[0] = Channel.of(
```

Comment:
I think you do not need the Channel.of statement. You can just give the inputs in `[ ]`

```
input[0] = [ [ id:'test' ],
file(params.modules_testdata_base_path + 'genomics/prokaryotes/bacteroides_fragilis/illumina/fasta/test1.contigs.fa.gz', checkIfExists: true)
]
```

Reply:
It is indeed not needed, but I like seeing input as Channels made the proper way